src/gui/src/displayControls.cpp

```diff
@@ -1651,7 +1651,7 @@ bool DisplayControls::areSelectedVisible()
 
 bool DisplayControls::isDetailedVisibility()
 {
-  return isModelRowVisible(&misc_.detailed);
+  return isModelRowVisible(&misc_.detailed) || isModelRowVisible(&misc_.module);
```

Comment:
I think the correct place to adjust this behavior would be in the renderthread.cpp. This causes the detailed view to be drawn even if the user has un-selected it.

Reply:
Could you rephrase/elaborate, I'm missing something that makes me not understand...

Reply:
without this do the memories get colored?
Looking at the code for drawing the module view: https://github.com/The-OpenROAD-Project/OpenROAD/blob/38a3b37e1a0bde4ef919c6a8bc4717516c3ad871/src/gui/src/renderThread.cpp#L1399C20-L1399C34
the instance list is controlled by: https://github.com/The-OpenROAD-Project/OpenROAD/blob/38a3b37e1a0bde4ef919c6a8bc4717516c3ad871/src/gui/src/renderThread.cpp#L1062
I think it would be cleaner to adjust the instance limit here, instead of changing the definition of detailed view.

Reply:
Hmm.... I'm out of my depth here. If we agree that this would be a nice feature, then it needs a 5 minute implementation by someone who knows what they are doing.

Of course nothing ever takes 5 minutes... but I'm thinking this is a small well defined task...

Reply:
I think it would be nice (I use the module view from time to time). I can take a stab at this sometime today